.github/workflows/ci_workflows.yml

```diff
@@ -17,6 +17,23 @@ jobs:
     name: Mandatory checks before CI
     runs-on: ubuntu-latest
     steps:
+    - name: Check base branch
+      uses: actions/github-script@v3
+      if: github.event_name == 'pull_request'
+      with:
+        script: |
+          const skip_label = 'skip-basebranch-check';
+          const allowed_basebranch = 'master';
```

Comment:
I don't know much about github-script, but I wonder if there is a way to retrieve the repository's default branch rather than hard-coding it (e.g. if we ever switch to "main").

Reply:
It looks like

```javascript
const { default_branch } = github.repos.get(context.repo);
```

would do it.

Reply:
I don't think this works. It says undefined...

Reply:
OK fixed it.

Reply:
Sorry for the misleading untested suggestion.  The docs on this were...not clear, and from the code it really looked like it should work :/  Thanks for figuring out the right answer.